internal/test/framework/framework.go

```diff
@@ -369,6 +384,100 @@ func (f *TestFramework) RetrieveArtifacts() {
 	}
 }
 
+// ApplyHybridOverlayPatch will enable the hybrid overlay on the cluster
+func (f *TestFramework) ApplyHybridOverlayPatch() error {
```

Comment:
nit: I am wondering if we want to specialize here with wsuFramework struct similar to what @suhanime is doing https://github.com/openshift/windows-machine-config-bootstrapper/pull/148. The main reason is ideally, we would want to take the `internal/framework` package and re-use it in our new wmco repo.